apollo-sample/src/main/java/com/example/apollostack/sample/DroidDetailsData.java

```diff
@@ -0,0 +1,26 @@
+package com.example.apollostack.sample;
+
+import com.google.auto.value.AutoValue;
+
+import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.Moshi;
+
+import javax.annotation.Nullable;
+
+import generatedIR.DroidDetails;
+
+@AutoValue
+public abstract class DroidDetailsData implements DroidDetails.Data {
+  @Override @Nullable public abstract Species species();
+
+  public static JsonAdapter<DroidDetailsData> jsonAdapter(Moshi moshi) {
+    return new AutoValue_DroidDetailsData.MoshiJsonAdapter(moshi);
+  }
+
+  @AutoValue
+  public abstract static class Species implements Specy {
```

Comment:
the pluralization lib erroneously singularized `species` to `specy`. I'm not sure but I think this word doesn't have a singular version